db/schema.rb

```diff
@@ -296,6 +296,7 @@
     t.datetime "updated_at",                                     null: false
     t.boolean  "completed",                      default: false
     t.datetime "follow_up_sent_at"
+    t.integer  "follow_up_count",      limit: 4, default: 0
   end
```

Comment:
Seems like we should also change the `follow_up_sent_at` to `last_follow_up_sent_at` since it can change.